xtask/src/ci.rs

```diff
@@ -31,6 +31,7 @@ impl CiTask {
             Some("msrv") => self.build_msrv()?,
             Some("stable") => self.build_stable()?,
             Some("nightly") => self.build_nightly()?,
+            Some("boo") => {}
```

Comment:
What happened here? :D

Reply:
Dangit I was so careful not to let this sneak in for so many commits noooooo. haha!